src/lib/server/cms.ts

```diff
@@ -29,6 +37,90 @@ purify.addHook('afterSanitizeAttributes', function (node) {
 		node.setAttribute('rel', 'noopener');
 	}
 });
+
+// Zod schema for external product data validation
+const externalProductSchema = z.object({
+	_id: z.string(),
```

Comment:
Why do we care about the `_id`? Shouldn't we simply use the URL of the resource?